### PR TITLE
funcRNA

### PR DESCRIPTION
**Pull request by jkobject**

### CODE REVIEW

**Comment by javadnoorb** [comment]
@jkobject  it looks like there's a merge conflict. Can you fix? -thanks

**Comment by jkobject** [comment]
done

File: depmapomics/postprocess_fusions.py [hunk 1]
```diff
@@ -1,18 +1,88 @@
...
+
+
+def renameFusionGene(a):
+  """
+  Given a fusion name from star-fusion, renames it
+  """
+  return [str(i.split('^')).replace(', ', ' (').replace("'", "")[1:-1]+')' for i in a]
+
+
 def standardize_leftgene_rightgene(fusions):
```
Comment (javadnoorb) [inline depmapomics/postprocess_fusions.py:76]: is this the same as the function on line 69?

**Review by javadnoorb — COMMENTED** [review]
I think there's one function which is repeated. Otherwise look good to me. I'm not able to see the changes in the notebook though so I didn't check that.

**Pull request merged**